nano/node/json_handler.cpp

```diff
@@ -956,17 +956,26 @@ void nano::json_handler::accounts_create ()
 void nano::json_handler::accounts_frontiers ()
 {
 	boost::property_tree::ptree frontiers;
-	auto transaction (node.store.tx_begin_read ());
-	for (auto & accounts : request.get_child ("accounts"))
+	auto transaction = node.store.tx_begin_read ();
+	for (auto & account_from_request : request.get_child ("accounts"))
```

Comment:
this is just one account got from the list of accounts `request.get_child ("accounts")`. The prefix `_from_request` is to state this is the account number got from the request. Notice that the request account number may differ from the decoded account. Example: if you decode `"nano_37rmg9h1iyyaqfdpmms3cm4fd5ewqdjcnj71i9ow14h3mx8dd5pn6ogqb1a1"` it will return `"nano_37rmg9h1iyyaqfdpmms3cm4fd5ewqdjcnj71i9ow14h3mx8dd5pn6ogqb1af"`. The last characters are corrected to `b1af`. To the response I think it is important to address what was requested, so it keeps the requested number in the response.